docs/setup/independent/install-kubeadm.md

```diff
@@ -125,6 +125,19 @@ systemctl enable docker && systemctl start docker
 
 {% endcapture %}
 
+**Note**: Please make sure the default cgroup driver for Docker is `systemd`. For certain versions of Docker,
+it will be set to `cgroupfs` which will cause incompatability issues with the kubelet. To fix, run this:
+
+```bash
+cat << EOF > /etc/docker/daemon.json
+{
+  "exec-opts": ["native.cgroupdriver=systemd"]
+}
+EOF
+```
+
+and restart Docker.
+
 {% assign tab_set_name = "docker_install" %}
```

Comment:
Suggestion: Move this note above, "You can use the following commands to install Docker."

Suggested changes to text:
Please make sure -> Make sure

For certain versions of Docker, it is set to `cgroupfs`, which causes incompatibility issues with the kubelet. To set the cgroup driver, run this command and then restart Docker:

Reply:
@kubernetes/sig-node-pr-reviews PTAL

Reply:
Technical wise, this is a little bit misleading. Users can do one of two things:

- config docker as suggested by this PR so docker will use  `systemd`
- config kubelet to use `cgroupfs`

With this PR, users may believe that the former way is the only solution.

Reply:
yeah, agreed. I updated the text. 

@steveperry-53 I couldn't shuffle because adding to the file assumes Docker has already been installed. Moving before the installation step wouldn't make sense IMO